### PR TITLE
PW-19 responsive design

### PR DESCRIPTION
**Pull request by KToldy**

### CODE REVIEW

**Review by DeMensa123 — APPROVED** [review]

**Review by MCFreddie777 — CHANGES_REQUESTED** [review]
image

Maybe we could make the heading and subheading smaller on the mobile devices, WDYT?

**Review by MCFreddie777 — COMMENTED** [review]
image

Also, could we stick to the design and center the menu?

File: src/views/Documents.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		class="bg-gray-200"
 		data-e2e-id="documentsPage"
 	>
-		<div class="container mx-auto mt-10">
```
Comment (MCFreddie777) [inline src/views/Documents.vue:6]: Table headers seems to be broken here.. we could decrease the width/padding/margin, couldn't we?

image

File: src/views/Contact.vue [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <template>
 	<div
-		class="bg-gray-100 flex items-center"
```
Comment (MCFreddie777) [inline src/views/Contact.vue:3]: The contact page seems different in the design. Could you please fix it?

image

image

**Review by MCFreddie777 — CHANGES_REQUESTED** [review]

**Comment by KToldy** [comment]
Fixes applied

File: src/views/Home.vue [hunk 3]
```diff
@@ -3,16 +3,16 @@
 		class="bg-primary"
 		data-e2e-id="homePage"
 	>
-		<div class="flex bg-gray-100 py-48 skewed-bottom-wrapper">
-			<div class="flex flex-col w-1/2 justify-center items-center">
-				<h1 class="font-bold text-6xl">
+		<div class="block lg:flex bg-gray-100 py-48 skewed-bottom-wrapper">
+			<div class="flex flex-col lg:w-1/2 justify-center items-center">
+				<h1 class="font-bold text-6xl ">
```
Comment (MCFreddie777) [inline src/views/Home.vue]: The heading still looks too big.

image
image

File: src/components/Navigation/NavigationItem.vue [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <template>
-	<li class="inline-block font-bold uppercase text-md hover:text-accent">
+	<li class="block mr-auto sm:mr-8 sm:inline-block font-bold uppercase text-md hover:text-accent">
```
Comment (MCFreddie777) [inline src/components/Navigation/NavigationItem.vue:2]: A little y-padding would come handy here. Looks too tight.

Also, could we add shadow to the whole wrapper and lighter primary to hover?

image


image

File: src/components/Document/DocumentTab.vue [hunk 5]
```diff
@@ -29,8 +29,6 @@ export default {
 	@apply mr-4;
```
Comment (MCFreddie777) [inline src/components/Document/DocumentTab.vue]: Still broken, you just took the padding/margin from outside of the whole container, not from the single item.

image

File: src/views/Contact.vue [hunk 6]
```diff
@@ -1,9 +1,9 @@
 <template>
```
Comment (MCFreddie777) [inline src/views/Contact.vue:1]: Okay, the skew looks good now but the heading is too small .

Also, can you smaller the icons in ContactItem? It breaks the words in the text in there.

image

image

**Review by MCFreddie777 — CHANGES_REQUESTED** [review]

File: src/components/Document/DocumentTab.vue [hunk 7]
```diff
@@ -29,8 +29,6 @@ export default {
...
 	margin-bottom: -2px;
 
-	&:first-child {
```
Comment (MCFreddie777) [inline src/components/Document/DocumentTab.vue:32]: Do not remove this please, you broke the first child. It was intentional.

**Review by MCFreddie777 — APPROVED** [review]

**Pull request merged**